fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -31,8 +31,12 @@ public static ConnectContext buildConnectContext() {
         // but QeProcessorImpl::reportExecStatus will check query id,
         // So we must disable report query status from BE to FE
         context.getSessionVariable().setReportSuccess(false);
-        // Always use 1 parallel to avoid affect normal query
-        context.getSessionVariable().setParallelExecInstanceNum(1);
+        int parallel = context.getSessionVariable().getStatisticCollectParallel();
+        if (null != ConnectContext.get()) {
+            parallel = ConnectContext.get().getSessionVariable().getStatisticCollectParallel();
```

Comment:
Why only parallel needs to judge ConnectContext

Reply:
Other variable don't need modify